packages/errors/src/errors.ts

```diff
@@ -1225,11 +1225,11 @@ export const AllCypressErrors = {
 
         If you have feedback about the experiment, please join the discussion here: http://on.cypress.io/single-tab-run-mode`
   },
-  EXPERIMENTAL_STUDIO_E2E_ONLY: () => {
+  EXPERIMENTAL_STUDIO_REMOVED: () => {
     return errTemplate`\
-        The ${fmt.highlight(`experimentalStudio`)} experiment is currently only supported for End to End Testing.
-
-        If you have feedback about the experiment, please join the discussion here: http://on.cypress.io/studio-beta`
+        The ${fmt.highlight(`experimentalStudio`)} was removed in ${fmt.cypressVersion(`15.4.0`)}.
```

Comment:
I think I would add that Studio is on by default here. We removed the feature at one point and there was a lot of backlash, so I don't want users to think it's gone as a feature.

```suggestion
        The ${fmt.highlight(`experimentalStudio`)} option was removed in ${fmt.cypressVersion(`15.4.0`)}.
        
        Cypress Studio is now available for all users.
```